boringssl 0.20241203.0

davidben opened the pull request.

bazel-io commented:
Hello @agl, @bob-beck, modules you maintain (boringssl) have been updated in this PR. Please review the changes.

bazel-io reviewed (dismissed):
Hello @bazelbuild/bcr-maintainers, all modules in this PR have been approved by their maintainers. Please take a final look to merge this PR.

davidben commented:
Build failures should be fixed on BoringSSL HEAD. We'll cut another tag with the fix and update this.

davidben commented:
Huh, I guess we should probably just remove `centos7_java11_devtoolset10` from the support matrix anyway, since CentOS 7 is EOL. google/oss-policies-info cites a minimum glibc version that doesn't include CentOS 7. Also we depend on googletest, which has also removed it from their matrix.

(We've also, I think, fixed the build issue, but I'll also remove it from the presubmit with the new tag, to match googletest.)

davidben commented:
Alright, updated. (Late because I forgot to add a reviewer for the new tag. Whoops!)

bazel-io reviewed (approved):
Hello @bazelbuild/bcr-maintainers, all modules in this PR have been approved by their maintainers. Please take a final look to merge this PR.
